SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,65 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Competition Details
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
```

Comment:
At SR2023 we had an unfortunate case of a team who were awarded a prize, but had left early and missed the prize-giving on the Sunday afternoon. We were unclear whether they had realised that they might get a prize and wanted to see if in future we could do more to advertise the prize-giving.

While this email has already been sent, I'm leaving this comment in the hope that for SR2025 we can consider highlighting more aspects of the schedule.